Add theme Arberia for Hugo

antedoro opened the pull request:
Add theme Arberia for Hugo

Demo site: https://arberiatheme.netlify.app/
Github Repo: https://github.com/antedoro/arberia

hugo-sid commented:
Thanks for adding Arberia theme.

As per [this](https://github.com/antedoro/arberia#license), your theme seems to be released under the GPLv2 license. Whereas the [LICENSE file](https://github.com/antedoro/arberia/blob/master/LICENSE) indicates it as MIT.

antedoro commented:
@hugo-sid Sorry for the mistake.

I have updated the README.
I hope now it's all ok.


> Thanks for adding Arberia theme.
> 
> As per [this](https://github.com/antedoro/arberia#license), your theme seems to be released under the GPLv2 license. Whereas the contents of [LICENSE file](https://github.com/antedoro/arberia/blob/master/LICENSE) seem to be as that of MIT.

Sorry for the mistake.

I have updated the README.
I hope now it's all ok.

hugo-sid commented:
No worries. Thanks for adding Arberia theme.
